### PR TITLE
Add Latest Commit Date Stat

### PR DESCRIPTION
**Pull request by mikeyoung85**

### CODE REVIEW

File: app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb [hunk 1]
```diff
@@ -22,6 +23,11 @@ def count_up_commits(dataset, since)
           return unless dataset.present?
           dataset.select {|commit| Date.parse(commit["date"]) >= since}.size
         end
+
+        def latest_commit(dataset)
+          latest = dataset&.sort_by{|commit| Date.parse(commit["date"])}&.reverse&.first
```
Comment (kbarrette) [inline app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb]: `reverse.first` == `last`?

Comment (kbarrette) [reply app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb]: What format is `date` in - it'd be nice to not have to parse it.

Comment (mikeyoung85) [reply app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb]: Will sorting work without Date parsing?

Comment (mikeyoung85) [reply app/models/maintenance_stats/stats/bitbucket/commit_count_stats.rb]: Dates are formatted like this: `2017-02-17T16:25:10+00:00`. I think thats the 8601 standard format.

**Comment by mikeyoung85** [comment]
Updated sort to not parse

**Pull request merged**